GameOfLife/Comunication/UserComunicator.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace GameOfLife
+{
+    /// <summary>
+    /// Manages comunication with user onput values
+    /// </summary>
+    public static class UserComunicator
```

Comment:
At first I thought it would be useful, to use static, because this class has a lot of references, but then it should use instances, since values in class change all the time and it is not really a static class.